packages/web/src/components/track/Artwork.tsx

```diff
@@ -121,6 +121,8 @@ const Artwork = memo(
       >
         {imageElement}
       </TrackFlair>
+    ) : (
+      imageElement
```

Comment:
when would it not be track?

Reply:
CollectionArtwork uses this too so i added the changes ^ to make doubly sure